dbt/adapters/trino/impl.py

```diff
@@ -21,6 +22,14 @@ class TrinoAdapter(SQLAdapter):
     ConnectionManager = TrinoConnectionManager
     AdapterSpecificConfigs = TrinoConfig
 
+    CONSTRAINT_SUPPORT = {
+        ConstraintType.check: ConstraintSupport.NOT_SUPPORTED,
+        ConstraintType.not_null: ConstraintSupport.NOT_SUPPORTED,
```

Comment:
If this works in iceberg, it would be nice to support this.

Reply:
I added new commit with support for `not-null` constraint. All tests needed to be marked with `@pytest.mark.iceberg` now, as catalog `memory` does not support non-null column